charts/grafana/templates/headless-service.yaml

```diff
@@ -20,4 +21,7 @@ spec:
   - protocol: TCP
     port: 3000
     targetPort: {{ .Values.service.targetPort }}
+    {{- with .Values.extraExposePorts }}
+    {{- tpl (toYaml . | nindent 2) $root }}
+    {{- end }}
```

Comment:
I don't think this will work. See:

https://github.com/clementnuss/helm-charts/blob/c79a5e44f21cc90832911c0ad7bf8105ed7b3623/charts/grafana/values.yaml#L198

Do you mean:
```yaml
  - protocol: TCP
    port: 3000
    targetPort: {{ .Values.service.targetPort }}
  {{- with .Values.extraExposePorts }}
  {{- tpl (toYaml . | nindent 2) $root }}
  {{- end }}
```